plugins/woocommerce-admin/client/customize-store/intro/index.tsx

```diff
@@ -125,6 +125,17 @@ export const Intro: CustomizeStoreComponent = ( { sendEvent, context } ) => {
 
 	const BannerComponent = BANNER_COMPONENTS[ bannerStatus ];
 
+	const sidebarMessage =
+		context.flowType === FlowType.AIOnline
+			? __(
+					'Create a store that reflects your brand and business. Select one of our professionally designed themes to customize, or create your own using AI',
+					'woocommerce'
+			  )
+			: __(
+					'Create a store that reflects your brand and business. Select one of our professionally designed themes to customize, or create your own using our store designer',
```

Comment:
Small detail: the provided copy expects a dot at the end of the sentence:

```suggestion
					'Create a store that reflects your brand and business. Select one of our professionally designed themes to customize, or create your own using our store designer.',
```